### PR TITLE
Even more section 3.8.1.1 updates

### PR DESCRIPTION
**Pull request by dericed**
I changed the wording of the recommendations of https://github.com/FFmpeg/FFV1/issues/258#issuecomment-782692779 in a few places, so please review critically and offer recommendations if the wording could be more clear.

### CODE REVIEW

File: ffv1.md [hunk 1]
```diff
@@ -443,11 +443,14 @@ R~i~
...
 r~i~
-: the boundary between two sub-ranges of R~i~: a sub-range of r~i~ values and a sub-range R~i~ - r~i~ values.
+: the Range at the i+1-th symbol. For R~i~ this represents the boundary between two sub-ranges of R~i~: a sub-range of r~i~ values and a sub-range R~i~ - r~i~ values.
 
 L~i~
 : the Low value of the Range at the i-th symbol.
 
+l~i~
+: the Low value of the Range at the i+1-th symbol.
+
```
Comment (michaelni) [inline ffv1.md:453]: This could be confusing. it would imply l() and L() are the same and redundant

Comment (dericed) [reply ffv1.md:453]: written as `a temporary variable to carry-over or adjust the Low value of the Range between range coding operations.`

**Comment by dericed** [comment]
updated based on a comment. Ready for re-review.

**Review by JeromeMartinez — APPROVED** [review]
LGTM.
I would remove " or writing" as we focus on reading, but not blocking.

**Review by retokromer — APPROVED** [review]

**Pull request merged**